diff --git a/plasma/plasmoids/src/sevenstart_src/src/CMakeLists.txt b/plasma/plasmoids/src/sevenstart_src/src/CMakeLists.txt
index c088890..9c857a0 100755
--- a/plasma/plasmoids/src/sevenstart_src/src/CMakeLists.txt
+++ b/plasma/plasmoids/src/sevenstart_src/src/CMakeLists.txt
@@ -14,7 +14,7 @@ target_link_libraries(io.gitgud.wackyideas.SevenStart
                       Qt6::Quick
                       KF6::I18n
                       KF6::WindowSystem
-                      Plasma
+                      Plasma::Plasma
                       PlasmaQuick
                       KF6::Svg
                       KF6::ConfigCore
@@ -23,9 +23,8 @@ target_link_libraries(io.gitgud.wackyideas.SevenStart
 
 
 #target_link_libraries(plasma_applet_seventasks KF5::WindowSystem)
-include_directories(/usr/include/Plasma /usr/include/KF6/KConfigCore/ /usr/include/KF6/KConfig/ /usr/include/KF6/KCoreAddons/)
 
 install(TARGETS io.gitgud.wackyideas.SevenStart DESTINATION ${KDE_INSTALL_PLUGINDIR}/plasma/applets)
 
 
-#plasma_install_package(package io.gitgud.wackyideas.SevenStart)
+plasma_install_package(package io.gitgud.wackyideas.SevenStart)
