diff --git a/CMakeLists.txt b/CMakeLists.txt
index a145603..e04f566 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ option(INSTALL_SERVICES "Install wait services" OFF)
 
 find_package(Plymouth REQUIRED)
 
-set(PLYMOUTH_THEME_INSTALL_DIR ${Plymouth_THEMESDIR})
+set(PLYMOUTH_THEME_INSTALL_DIR ${Plymouth_THEMESDIR} CACHE STRING "Where to install the Plymouth theme")
 
 set(BOOT_ANIM_START_DELAY "2"    CACHE STRING "Time in seconds before the boot animation begins")
 set(BOOT_PROGRESS_CLEAR   "0.99" CACHE STRING "Value between 0 and 1 to clear the entire screen")
