diff --git a/src/sycoca/vfolder_menu.cpp b/src/sycoca/vfolder_menu.cpp
index 12313911..bda71e35 100644
--- a/src/sycoca/vfolder_menu.cpp
+++ b/src/sycoca/vfolder_menu.cpp
@@ -398,12 +398,13 @@ QString VFolderMenu::absoluteDir(const QString &_dir, const QString &baseDir, bo
         dir = QStandardPaths::locate(QStandardPaths::GenericConfigLocation, QLatin1String("menus/") + dir, QStandardPaths::LocateDirectory);
     }
 
-    if (!relative) {
-        QString resolved = QDir(dir).canonicalPath();
-        if (!resolved.isEmpty()) {
-            dir = resolved;
-        }
-    }
+    // do resolve symlinks because nix
+    // if (!relative) {
+    //     QString resolved = QDir(dir).canonicalPath();
+    //     if (!resolved.isEmpty()) {
+    //         dir = resolved;
+    //     }
+    // }
 
     if (!dir.endsWith(QLatin1Char('/'))) {
         dir += QLatin1Char('/');
